projects/clearpool/index.js

```diff
@@ -11,44 +11,120 @@ const CPOOL = "0x66761fa41377003622aee3c7675fc7b5c1c2fac5";
 
 module.exports = {
   methodology: "We count liquidity by Stables deposited on the pools contracts",
-}
+};
+
+const CHAIN = {
+  ARBITRUM: "arbitrum",
+  AVAX: "avax",
+  ETHEREUM: "ethereum",
+  OPTIMISM: "optimism",
+  POLYGON: "polygon",
+  POLYGON_ZKEVM: "polygon_zkevm",
+  FLARE: "flare",
+  BASE: "base"
+};
```

Comment:
is it possible to use directly the `sdk` package without declaring internally?

Reply:
@Addis-Belete How about `Mantle`?

Reply:
@georgeciubotaru No, the networks are listed in the helper/chains as an array.